src/components/indicators.js

```diff
@@ -1,29 +1,33 @@
 // @flow
 import React, { type Node } from 'react';
-import { type Emotion } from 'create-emotion';
+import { ClassNames, keyframes } from '@emotion/core';
 
 import type { CommonProps, Theme } from '../types';
 
 // ==============================
 // Dropdown & Clear Icons
 // ==============================
 
-const Svg = ({ size, emotion, ...props }: { size: number, emotion: Emotion }) => (
-  <svg
-    height={size}
-    width={size}
-    viewBox="0 0 20 20"
-    aria-hidden="true"
-    focusable="false"
-    className={emotion.css({
-      display: 'inline-block',
-      fill: 'currentColor',
-      lineHeight: 1,
-      stroke: 'currentColor',
-      strokeWidth: 0,
-    })}
-    {...props}
-  />
+const Svg = ({ size, ...props }: { size: number }) => (
+  <ClassNames>
+    {({ css }) => (
+      <svg
+        height={size}
+        width={size}
+        viewBox="0 0 20 20"
+        aria-hidden="true"
+        focusable="false"
+        className={css({
```

Comment:
@mitchellhamilton We technically don't need to be using ClassNames here to access the css method, since we aren't generating a custom className using our own cx implementation, however I noticed that using the css prop with the jsx pragma generated a style tag without a nonce attribute. Not sure why this is the case, but using the ClassNames renderProp to circumnavigate this for now

Reply:
That's very strange, gonna look into it.